consensus/polybft/consensus_runtime.go

```diff
@@ -231,6 +236,10 @@ func (c *consensusRuntime) populateFsmIfBridgeEnabled(
 	}
 
 	if isEndOfSprint {
+		if err != nil {
```

Comment:
This error check should be removed

Reply:
Fixed [43050c7](https://github.com/0xPolygon/polygon-edge/pull/796/commits/43050c73e6a58bdc050ef8e7e0734e30b81072f8)